index.bs

```diff
@@ -438,4 +439,4 @@ A file containing a 'pict' track compliant with this profile is expected to list
 <h2 id="change-list">Changes since v1.1.0 release</h2>
     - EDITORIAL: <a href="https://github.com/AOMediaCodec/av1-avif/pull/223">Stop using `dfn value` for definitions.</a>
     - EDITORIAL: <a href="https://github.com/AOMediaCodec/av1-avif/pull/222">Add assert-ids in the spec for conformance file testing and ComplianceWarden</a>
-
+    - <a href="https://github.com/AOMediaCodec/av1-avif/pull/234">Fix typo in av1C regarding metadata OBUs</a>
```

Comment:
What was the typo? `PixelInformationProperty or ColourInformationBox` instead of `MasteringDisplayColourVolumeBox or ContentLightLevelBox`?

Should this line mention the new equality constraint between `av1C` bit depth and `pixi` bit depth?

Reply:
The "typo" (per issue #156) was that bit depth is not really part of the metadata OBUs. I guess I should really reword the change entry as "Clarify relationship between av1C, metadata OBUs and item properties".